egs/librispeech_100/asr1/conf/tuning/train_conformer_transducer.yaml

```diff
@@ -0,0 +1,55 @@
+# minibatch related
+batch-size: 16
+maxlen-in: 512
+maxlen-out: 150
+
+# optimization related
+criterion: loss
+early-stop-criterion: "validation/main/loss"
+sortagrad: 0
+opt: noam
+transformer-lr: 5.0
+transformer-warmup-steps: 25000
+epochs: 100
+patience: 0
+accum-grad: 8
+grad-clip: 5.0
+
+# network architecture
+## general
+custom-enc-positional-encoding-type: rel_pos
+custom-enc-self-attn-type: rel_self_attn
+custom-enc-pw-activation-type: swish
+## encoder related
+etype: custom
+custom-enc-input-layer: vgg2l
+enc-block-arch:
+        - type: conformer
+          d_hidden: 256
+          d_ff: 1024
+          heads: 4
+          macaron_style: True
+          use_conv_mod: True
+          conv_mod_kernel: 15
+          dropout-rate: 0.3
+          att-dropout-rate: 0.3
+enc-block-repeat: 18
+## decoder related
+dtype: lstm
+dlayers: 1
+dec-embed-dim: 300
+dunits: 300
+dropout-rate-embed-decoder: 0.2
+dropout-rate-decoder: 0.1
+## joint network related
+joint-dim: 300
+
+# transducer related
+model-module: "espnet.nets.pytorch_backend.e2e_asr_transducer:E2E"
+
+# reporter related
+report-wer: True
+report-cer: True
```

Comment:
I would suggest setting them to `False` by default. If we are sure about the training, it's better to save the users some time.

Reply:
Thanks! I'll set them to False. But I'll keep them True for CTC models since they need to be reported for averaging models with the validation CERs.